cmd/collector/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+EXPOSE 14267
+
+COPY collector-linux /go/bin/
+
+CMD ["/go/bin/collector-linux"]
```

Comment:
This comment applies to all images - don't we need to be translating env vars into command line switches, to make the images follow https://12factor.net/ ?

Ideally our binaries could be reading env vars directly, there are frameworks like spf13/Cobra that support that, but we haven't implemented it yet. So the only way currently to configure each binary is to translate env vars into command line switches.